### PR TITLE
Docs: add "Edit this page in Github" link

### PR DESCRIPTION
**Pull request by jhonnyoddball**
## Description
See ticket [#9533](https://github.com/department-of-veterans-affairs/va.gov-team/issues/9533) for description

## Testing done
Locally

## Screenshots

Screen Shot 2020-07-07 at 12 45 39 PM

## Acceptance criteria
- [x] Add "Edit this page in GitHub" 

## Definition of done
- [ ] Changes have been tested in vets-website
- [ ] Changes have been tested in IE11, if applicable
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: packages/documentation/src/components/pageFooter/Documentation.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        : null;
+    });
+
+    const level2 = level2Sections.map((section) => {
+      if (section && section.href && section.href === location.pathname) {
+        return section && section.indexmdx ? true : false;
+      }
+    });
+
+    if (level2.find((indexmdx) => indexmdx)) {
```
Comment (cvalarida) [inline packages/documentation/src/components/pageFooter/Documentation.jsx:34]: Can `level2` ever be something other than an array?

Comment (cvalarida) [inline packages/documentation/src/components/pageFooter/Documentation.jsx:34]: I'm not following this 100%. Is the goal to find the first truthy value in `level2`, or is the goal to find the first object where `{ indexmdx: true }`?

File: packages/documentation/src/components/pageFooter/Documentation.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  if (sidebarSection?.items) {
+    const level2Sections = sidebarSection.items.map((item) => {
+      return item.items
+        ? item.items.find((lvl2Item) => lvl2Item.href === location.pathname)
+        : null;
+    });
+
+    const level2 = level2Sections.map((section) => {
+      if (section && section.href && section.href === location.pathname) {
+        return section && section.indexmdx ? true : false;
```
Comment (cvalarida) [inline packages/documentation/src/components/pageFooter/Documentation.jsx:30]:
```suggestion
        return section?.indexmdx ? true : false;
```

File: packages/documentation/src/components/pageFooter/Documentation.jsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  // Test second Level of the sidebar object
+  if (sidebarSection?.items) {
+    const level2Sections = sidebarSection.items.map((item) => {
+      return item.items
+        ? item.items.find((lvl2Item) => lvl2Item.href === location.pathname)
+        : null;
+    });
+
+    const level2 = level2Sections.map((section) => {
+      if (section && section.href && section.href === location.pathname) {
```
Comment (cvalarida) [inline packages/documentation/src/components/pageFooter/Documentation.jsx:29]:
```suggestion
      if (section?.href === location.pathname) {
```

File: packages/documentation/src/components/pageFooter/Documentation.jsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  );
+
+  if (level1 && level1.indexmdx) {
+    return '/index';
+  }
+
+  // Test second Level of the sidebar object
+  if (sidebarSection?.items) {
+    const level2Sections = sidebarSection.items.map((item) => {
+      return item.items
```
Comment (cvalarida) [inline packages/documentation/src/components/pageFooter/Documentation.jsx:23]:
```suggestion
      return Array.isArray(item.items)
```

File: packages/documentation/src/components/pageFooter/Components.jsx [hunk 5]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Link, StaticQuery, graphql } from 'gatsby';
+
+/**
+ * Sidebar
+ *
+ * @param {props} props
```
Comment (cvalarida) [inline packages/documentation/src/components/pageFooter/Components.jsx:7]: Should this be an object?
```suggestion
 * @param {object} props
```
Also, you can specify properties on the object to provide more clarity:
```
@param {string} thing.propertyOfThing
```

File: packages/documentation/src/sidebar.js [hunk 6]
```diff
@@ -4,6 +4,7 @@ module.exports = {
       id: 'getting-started',
       name: 'Getting started',
       href: '/getting-started',
+      indexmdx: true,
```
[the remark this answers is not in the document]
Comment (jhonnyoddball) [reply packages/documentation/src/sidebar.js:7]: It's not always the case and for that reason I have to add this flag
```
            {
              name: 'VA.gov environments',
              href: '/platform/site-structure/environments',
              indexmdx: true,
            },
```

File: packages/documentation/src/components/pageFooter/Documentation.jsx (hunk 1 above)
Comment (jhonnyoddball) [reply packages/documentation/src/components/pageFooter/Documentation.jsx:34]: Correct!
The goal to find the first object where `{ indexmdx: true }`

**Pull request merged**